astropy/io/fits/column.py

```diff
@@ -1119,7 +1119,9 @@ def _verify_keywords(cls, name=None, format=None, unit=None, null=None,
                     "of formatting this column.")
 
             if dims_tuple:
-                if reduce(operator.mul, dims_tuple) > format.repeat:
+                if isinstance(recformat, _FormatP):
+                    msg = None
```

Comment:
This is to avoid the warning for P and Q format as TDIM has a different meaning in VLA formats